ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -142,25 +229,99 @@ export const CoinOverview = ({
           <div className={`${classPrefix}-overview__balance`}>
             <div className={`${classPrefix}-overview__primary-container`}>
               {balanceToDisplay ? (
-                <UserPreferencedCurrencyDisplay
-                  style={{ display: 'contents' }}
-                  className={classnames(
-                    `${classPrefix}-overview__primary-balance`,
-                    {
-                      [`${classPrefix}-overview__cached-balance`]:
-                        balanceIsCached,
-                    },
-                  )}
-                  data-testid={`${classPrefix}-overview__primary-currency`}
-                  value={balanceToDisplay}
-                  type={PRIMARY}
-                  ethNumberOfDecimals={4}
-                  hideTitle
-                  shouldCheckShowNativeToken
-                  isAggregatedFiatOverviewBalance={
-                    !showNativeTokenAsMainBalance
-                  }
-                />
+                <>
+                  <Box onMouseEnter={handleMouseEnter} ref={setBoxRef}>
+                    <UserPreferencedCurrencyDisplay
+                      style={{ display: 'contents' }}
+                      className={classnames(
+                        `${classPrefix}-overview__primary-balance`,
+                        {
+                          [`${classPrefix}-overview__cached-balance`]:
+                            balanceIsCached,
+                        },
+                      )}
+                      data-testid={`${classPrefix}-overview__primary-currency`}
+                      value={balanceToDisplay}
+                      type={PRIMARY}
+                      ethNumberOfDecimals={4}
+                      hideTitle
+                      shouldCheckShowNativeToken
+                      isAggregatedFiatOverviewBalance={
+                        !showNativeTokenAsMainBalance && !isTestnet
+                      }
+                    />
+                  </Box>
+                  {shouldShowPopover &&
+                  !isTestnet &&
+                  !showNativeTokenAsMainBalance ? (
+                    <Popover
+                      referenceElement={referenceElement}
+                      isOpen={isOpen}
+                      position={PopoverPosition.BottomStart} // TODO check with design-team about this bottom start issue
+                      hasArrow
+                      flip
+                      backgroundColor={BackgroundColor.overlayAlternative} // TODO check with DS on this opacity issue
```

Comment:
Hello again @georgewrmarshall 👋 
We also have this color in [figma](https://www.figma.com/design/aMYisczaJyEsYl1TYdcPUL/Portfolio-View?node-id=4098-126757&node-type=instance&focus-id=4186-130770&m=dev) that looks a bit darker than the 
`backgroundColor={BackgroundColor.overlayAlternative} `

Sorry, I wasn't quite sure about the solution you suggested 🙏

Reply:
Thank you @georgewrmarshall!
That was the trick 🙏 
Looks good now

cc @amandaye0h 

<img width="356" alt="Screenshot 2024-09-16 at 23 59 47" src="https://github.com/user-attachments/assets/93a82e4c-cd72-4da0-92a2-748f7dc7de7c">

<img width="364" alt="Screenshot 2024-09-16 at 23 58 57" src="https://github.com/user-attachments/assets/05109992-fedd-43b8-9a55-2dbc8e864c55">